recipes/medcoupling/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "medcoupling" %}
+{% set version = "9.10.0" %}
+
+
+package:
+  name: '{{ name|lower }}'
```

```suggestion
  name: medcoupling
```

I think the whole jinja name variable is overused, it leads to too much confusion for no advantage.

---

Yeah, I see your point. Will update